jose/Utils.ml

```diff
@@ -20,6 +20,11 @@ module RResult = struct
     | Ok a, Ok b, Ok c, Ok d, Ok e, Ok f, Ok g, Ok h ->
         Ok (a, b, c, d, e, f, g, h)
     | _ -> Error (`Msg "all 8 was not Ok")
+
+  let get_exn = function
+    | Ok a -> a
+    | Error (`Msg m) -> raise (Invalid_argument m)
```

Comment:
⛳: use the public API `invalid_arg m`

Reply:
get_exn is now removed since I don't want to  use it when released, but I will keep this in mind for future hacks/spikes